.travis.yml

```diff
@@ -1,3 +1,6 @@
+sudo: required
```

Comment:
Thanks. Then I think this is fine, but I'm tired, so I'll leave it for others to have a look at in case I'm overlooking something.